src/pages/fr/reference/configuration-reference.md

```diff
@@ -0,0 +1,507 @@
+
+:::caution
+Fournir une liste de plugins va **retirer** nos plugins par défaut. Pour les préserver, regardez l'option `extendDefaultPlugins`.
+:::
+
+```js
+import rehypeMinifyHtml from 'rehype-minify';
+{
+  markdown: {
+    rehypePlugins: [rehypeMinifyHtml]
+  }
+}
+```
+
+
+### markdown.extendDefaultPlugins
+
+<p>
+
+**Type :** `boolean`<br>
+**Par défaut :** `false`
+</p>
+
+Astro applique par défaut les plugins [Markdown de GitHub](https://github.com/remarkjs/remark-gfm) et [Smartypants](https://github.com/silvenon/remark-smartypants). Quand vous ajoutez votre propre plugin remark ou rehype, vous pouvez préserver ces plugins par défaut en paramétrant l'option `extendDefaultPlugins` sur `true` 
+
+```js
+{
+  markdown: {
+    extendDefaultPlugins: true,
+		 remarkPlugins: [exampleRemarkPlugin],
+    rehypePlugins: [exampleRehypePlugin],
+  }
+}
+```
+
+
+### markdown.remarkRehype
+
+<p>
+
+**Type:** `RemarkRehype`
+</p>
+
+Passez les options à [remark-rehype](https://github.com/remarkjs/remark-rehype#api).
+
+```js
+{
+  markdown: {
+    // Exemple: Traduire le texte du footer dans un autre langage, ce sont les valeurs anglaises par défaut.
+    remarkRehype: { footnoteLabel: "Footnotes", footnoteBackLabel: "Back to content"},
+  },
+};
+```
+
+
+## Intégrations
+
+Étendez Astro avec des intégrations personnalisées. Les intégrations sont votre guichet unique pour ajouter la prise en charge du framework (comme Solid.js), de nouvelles fonctionnalités (comme les sitemaps) et de nouvelles bibliothèques (comme Partytown et Turbolinks).
+
+Lisez notre [Guide d'intégration](/fr/guides/integrations-guide/) pour vous aider à débuter avec les intégrations Astro.
+
+```js
+import react from '@astrojs/react';
+import tailwind from '@astrojs/tailwind';
+{
+  // Exemple: Ajouter la prise en charge de React + Tailwind par Astro
+  integrations: [react(), tailwind()]
+}
+```
+
+## Vite
+
+Passez des options de configuration supplémentaires à Vite. Utile lorsque Astro ne prend pas en charge certaines configurations avancées dont vous pourriez avoir besoin.
+
+Regardez la documentation complète de l'objet de configuration de `vite` sur [vitejs.dev](https://vitejs.dev/config/).
+
+#### Exemples
+
+```js
+{
+  vite: {
+    ssr: {
+      // Exemple: Forcer un package cassé à passer outre le processus SSR si besoin.
+      external: ['broken-npm-package'],
+    }
+  }
+}
+```
+
+```js
+{
+  vite: {
+    // Exemple: Ajout des plugins personnalisés vite directement à votre projet Astro.
+    plugins: [myPlugin()],
+  }
+}
+```
+
+## Legacy Flags
+
+Pour aider certains utilisateurs à migrer entre les versions d’Astro, nous introduisons occasionnellement des drapeaux `legacy`.
+Ces drapeaux vous permettent d’opter pour un comportement déprécié ou autrement dépassé d'Astro
+dans la dernière version, afin que vous puissiez continuer à mettre à jour et profiter des nouvelles versions d’Astro.
+
```

Comment:
In this context “flag” in English actually means “option”. Here’s the section of French Wikipedia that explains this:

> Une option en ligne de commande ou plus une option (que l’on appelle aussi parfois un flag) modifie le fonctionnement d'une commande.
> <cite>https://fr.wikipedia.org/wiki/Interface_en_ligne_de_commande#Option</cite>

Maybe “Options dépassés” or something?

Reply:
Yes, thank you.